src/workers/job-processes.js

```diff
@@ -42,6 +42,18 @@ export async function processJobs() {
   }
 }
 
+export async function checkMessageQueue() {
+  console.log('checking if messages are in message queue');
```

Comment:
To stop continually running something that is only used during a setting, let's add the same
```
if(!process.env.TWILIO_SQS_QUEUE_URL) {
   return
}
```
to the top of the function here, so we don't loop at all when the setting isn't there.